README.md

```diff
@@ -27,14 +27,17 @@ Refer to the [`examples`](examples) folder for the source code of these examples
 
 ### Minimal
 
-```js
-const { MailtrapClient } = require("mailtrap");
+```ts
+import { MailtrapClient } from "../src"
```

Comment:
I don't get this change or maybe I missed something. This an example for a client, how is `MailtrapClient` going to be available in clients' `../src` folder?